### PR TITLE
ci: add ci/skip-test label to upgrade PR and better message

### PR DESCRIPTION
**Pull request by lahabana**
This includes a link to the repo which is always useful

Signed-off-by: Charly Molter <charly.molter@konghq.com>

### CODE REVIEW

**Review by michaelbeaumont — APPROVED** [review]

File: .github/workflows/create-gui-pr.yml [hunk 1]
```diff
@@ -75,19 +75,16 @@ jobs:
...
-
-            [skip ci]
+            Bumps ${{ github.repository }} to version [${{ github.ref_name }}@${{ github.sha }}](https://github.com/${{ github.repository }}/tree/${{ github.sha }})
           committer: GitHub <noreply@github.com>
           author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
           signoff: true
           branch: chore/update-gui
           delete-branch: true
-          title: "chore(deps): bump kuma-gui"
+          title: "chore(deps): bump ${{ github.repository }} to ${{ github.sha }}"
```
Comment (kleinfreund) [inline .github/workflows/create-gui-pr.yml:86]: **Suggestion**: I think this should not refer to the git hash

```suggestion
          title: "chore(deps): bump ${{ github.repository }}"
```

I’m not sure whether the action would update the title of an existing PR, but in any case, I think it’s too verbose here.

**Review by kleinfreund — APPROVED** [review]

File: .github/workflows/create-gui-pr.yml (hunk 1 above)
Comment (michaelbeaumont) [reply .github/workflows/create-gui-pr.yml:86]: I kind of agree, though I see the potential use of being able to search for the PR that updated to some hash

Comment (lahabana) [reply .github/workflows/create-gui-pr.yml:86]: It does update the title.

Moreover the PR title is what we use as first of git commit see: https://github.com/kumahq/kuma/commit/6b22e376f41d02665fd891366e04720ca995f6a9

Comment (kleinfreund) [reply .github/workflows/create-gui-pr.yml:86]: I don’t feel strongly about this so I’m happy to keep it.

**Pull request merged**